packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -0,0 +1,7 @@
+:local {
+  .embedded-content-container {
+    display: flex;
+    flex-direction: column;
+    height: auto;
```

Comment:
Do we want the height to be 100% of its container?  If we set the height of this container, do we still require the flex box layout?  If not, I will approve it.  I think there might be a layout issue somewhere since it is does not seem like we are setting the height and width on our container elements.

Reply:
I could not set the height here without the flexbox layout but I updated this to set the height of the VisuallyHiddenText to 0 instead: https://github.com/cerner/terra-framework/pull/1804/commits/96b994e2a096fc71aac6376fd3b0760e1b41f171